src/io/CavityWriter.h

```diff
@@ -30,11 +47,18 @@ class CavityWriter : public PluginBase {
 		return std::string("CavityWriter");
 	}
 	static PluginBase* createInstance() { return new CavityWriter(); }
+
+	std::map<unsigned, CavityEnsemble*> getMcav(){ return _mcav;}
 private:
 	std::string _outputPrefix;
 	unsigned long _writeFrequency;
 	bool _appendTimestamp;
 	bool _incremental;
+	int _Nx = 0, _Ny = 0, _Nz = 0;
+    int _maxNeighbors = 0;
```

Comment:
formatting

Reply:
Not present in my IDE

Reply:
that's a problem of difference between indentation by tab and indentations by space. please. make sure that everything in the two files is indented by tabs.